virtool/samples/files.py

```diff
@@ -63,28 +66,33 @@ async def create_artifact_file(
 
 
 async def create_reads_file(
```

Comment:
This function is becoming overly complex. A function should ideally do one thing well. Keep this function SQL-focussed.

I suggest writing another function in `virtool.samples.fake` dedicated to copying the example reads to the sample directory. Call this in `create_fake_sample()` before creating a SQL record for the reads file using `create_reads_file()`.